src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -287,13 +287,7 @@ void ExitCheckOverflow()
                 "Regex.InfiniteMatchTimeout" :
                 $"TimeSpan.FromMilliseconds({matchTimeout.ToString(CultureInfo.InvariantCulture)})";
 
-        /// <summary>Adds the IsWordChar helper to the required helpers collection.</summary>
-        private static void AddIsWordCharHelper(Dictionary<string, string[]> requiredHelpers)
-        {
-            const string IsWordChar = nameof(IsWordChar);
-            if (!requiredHelpers.ContainsKey(IsWordChar))
-            {
-                requiredHelpers.Add(IsWordChar, new string[]
+        private static readonly string[] s_isWordChar = new string[]
```

Comment:
Please revert the changes in this file and just suppress the analyzer for the file instead.